patches/content-browser-renderer_host-render_frame_host_impl.cc.patch

```diff
@@ -0,0 +1,53 @@
+diff --git a/content/browser/renderer_host/render_frame_host_impl.cc b/content/browser/renderer_host/render_frame_host_impl.cc
+index e4e16ccdcd2070e56d9572b7d4bc22d685cb34bc..a082718218cc220179b88672fddaa1a591006c73 100644
+--- a/content/browser/renderer_host/render_frame_host_impl.cc
++++ b/content/browser/renderer_host/render_frame_host_impl.cc
+@@ -7951,6 +7951,48 @@ void RenderFrameHostImpl::BeginNavigation(
+   if (!VerifyBeginNavigationCommonParams(GetSiteInstance(), &*validated_params))
+     return;
+ 
++  GURL url = validated_params->url;
```

Comment:
This patch is used for renderer-initiated navigations (e.g. clicking on a link).